docs/docusaurus/docs/core/expectations/manage_expectations.md

```diff
@@ -0,0 +1,140 @@
+  ```python name="core/expectations/_examples/edit_an_expectation.py get expectation"
+  ```
+
+2. Modify the Expectation's attributes.
+  ```python name="core/expectations/_examples/edit_an_expectation.py modify attributes"
+  ```
+  The specific attributes that can be modified correspond to the parameters used to initialize the Expectation.  You can view available Expectations and the parameters they take in the [Expectation Gallery](https://greatexpectations.io/expectations).
+
+3. (Optional) If the Expectation belongs to an Expectation Suite, save the changes to the Expectation Suite.
+  ```python name="core/expectations/_examples/edit_an_expectation.py save the Expectation"
+  ```
+  :::info
+  `expectation.save()` is explicitly used to update the configuration of an Expectation in an Expectation Suite.  If the Expectation is not part of an Expectation Suite, `expectation.save()` will fail.
+  :::
+
+<details><summary>Full example code</summary>
+<p>
+
+```python name="core/expectations/_examples/edit_an_expectation.py full example code
+```
+
+</p>
+</details>
+
+## Customize an Expectation Class
+
+1. Choose and import a base Expectation class.
+
+  You can view available Expectations in the [Expectation Gallery](https://greatexpectations.io/expectations).  This example will create a customized Expectation for a valid passenger count in taxi data out of the base Expectation `ExpectColumnValueToBeBetween`.
+
+  ```python
+from great_expectations.expectations import ExpectColumnValueToBeBetween
+  ```
+
+
+2. Create a new Expectation that inheirits the base Expectation class.
+
+ ```python
+class ExpectValidPassengerCount(ExpectColumnValueToBeBetween):
+ ```
+
+3. Override the Expectation's attributes with new default values.
+
+  ```python
+class ExpectValidPassengerCount(ExpectColumnValueToBeBetween):
+    column: str = "passenger_count"
+    min_value: int = 0
+    max_value: int = 6
+  ```
+
+  The attributes that can be overriden correspond to the parameters required by the base Expectation.  These can be referenced from the [Expectation Gallery](https://greatexpectations.io/expectations).
+
+4. Customize the rendering of the new Expectation when displayed in Data Docs.
+
+  The `render_text` attribute contains the text describing the customized Expectation when your results are rendered into Data Docs.  This text can be formatted with Markdown syntax.
+
+  ```python
+class ExpectValidPassengerCount(ExpectColumnValueToBeBetween):
+    column: str = "passenger_count"
+    min_value: int = 0
+    max_value: int = 6
+    render_text: str = "There should be between **0** and **6** passengers."
+  ```
+  
+
+
+## Next steps
```

Comment:
Are these next steps, or related documentation?

Reply:
These will become "Next steps" -- for now they're a bit of a place holder, since those documents are going to undergo revision from GX OSS to GX Core.  I'll be going through all of the guides and putting in appropriate "Next steps" as part of a future PR.